content/understanding-urbit/history-people.md

```diff
@@ -0,0 +1,125 @@
++++
+title = "People + History"
+description = "Where we've been and were we are now"
+weight = 5
+[extra]
+flatten_pagination = "true"
+hide_next_title = "true"
+hide_previous_title = "true"
+image = "https://media.urbit.org/site/understanding-urbit/project-history/project-status-landscape-earth%402x.png"
++++
+
+So who builds this thing? Where does it come from? What’s the history? 
+
+Today, Urbit is built and maintained by two companies, a community of contributors and core developers, infrastructure (star) operators, and governance (galaxy) node owners. 
+
+Let’s talk first about these individual groups within the Urbit community, then about the history of the project and how it came to be.
+
+![](https://media.urbit.org/site/understanding-urbit/your-last-computer/your-last-computer-waves%402x.png)
+
+There are currently two independent companies that work on building Urbit full-time: [Tlon](https://tlon.io/) and [urbit.live](https://urbit.live/). 
+
+Tlon has been around since 2013 or so, and is primarily focussed on the development of Urbit OS and its user experience. Urbit.live was started in 2018 to make the onramp into Urbit as easy as possible. Both companies contribute regularly to the code base.
+
+Tlon was originally granted about half of the Urbit ID address space, and has used that to fund its operations so far. Over time, Tlon plans to offer hosting and services to make using Urbit as easy as possible. 
+
+Tlon is also working to set up an urbit.org foundation. This is still in process, and is expected to happen in 2020.
+
+Urbit.live started selling Urbit IDs very early once that became possible, and also plans to offer services within the Urbit ecosystem. 
+
+The community of contributors and core developers has been steadily contributing to the code base since before Tlon existed, and they’re still going strong. They can primarily be found on Urbit itself, in `~dopzod/urbit-help`, the [urbit-dev mailing list](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&id=be143888d2) and on [GitHub](https://github.com/urbit/urbit/issues).
```

Comment:
urbit-dev is [here](https://groups.google.com/a/urbit.org/forum/#!forum/dev).

GitHub should link to just urbit/urbit.

Reply:
The urbit-dev link is still pointing to Mailchimp, not to the Google group.